scan/spec/error_handler_spec.rb

```diff
@@ -7,19 +7,21 @@
     describe "handle_build_error" do
       describe "when parsing parallel test failure output" do
         it "does not report a build failure" do
+          expect(Scan).to receive(:config).and_return({})
           output = File.open('./scan/spec/fixtures/parallel_testing_failure.log', &:read)
           expect do
             Scan::ErrorHandler.handle_build_error(output, log_path)
-          end.to_not(raise_error(FastlaneCore::Interface::FastlaneBuildFailure))
+          end.not_to raise_error
         end
       end
 
       describe "when parsing non-parallel test failure output" do
         it "does not report a build failure" do
+          expect(Scan).to receive(:config).and_return({})
           output = File.open('./scan/spec/fixtures/non_parallel_testing_failure.log', &:read)
           expect do
             Scan::ErrorHandler.handle_build_error(output, log_path)
-          end.to_not(raise_error(FastlaneCore::Interface::FastlaneBuildFailure))
+          end.to_not(raise_error)
```

Comment:
Line 14 says `not_to raise_error` and this one `to_not(raise_error)`, can we standardize? :)

Reply:
We use both (130+ vs 190+).

Maybe I do this as a separate PR.